templates-pre/index.html

```diff
@@ -1,3 +1,5 @@
+{% set svgName = "all-sprt" %}
+{% set pagetitlehere = "Home" %}
```

Comment:
Can you please change this variable name to something relevant?
Like `pageTitle`.